src/loaders/common/loader_helper.py

```diff
@@ -77,3 +84,53 @@ def init_genome_atrri_doc(kbase_collection, load_version, genome_id):
     }
 
     return doc
+
+
+def get_token(token_filename):
+    """
+    Get token if a filename is provided at home dir.
+    """
+    try:
+        home_dir = os.path.expanduser("~")
+        with open(os.path.join(home_dir, "." + token_filename), "r") as f:
+            token = f.readline().strip()
+    except FileNotFoundError as e:
+        raise e
```

Comment:
No need for the try / catch if the same exception is being raised

Reply:
Updated above!

Reply:
:+1: